src/redux/store.js

```diff
@@ -0,0 +1,15 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux'
+
+import dataReducer from './data/dataReducer'
+import userInputReducer from './userInput/userInputReducer'
+
+const thunkMiddleware = require('redux-thunk').default
+
+const rootReducer = combineReducers({
```

Comment:
va aggiunto il supporto ai devtools redux https://redux.js.org/tutorials/fundamentals/part-4-store#adding-the-devtools-to-the-store